src/odb/BUILD

```diff
@@ -0,0 +1,113 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2025-2025, Precision Innovations Inc.
+
+load("//:bazel/tcl_encode_or.bzl", "tcl_encode")
+load("//:bazel/tcl_wrap_cc.bzl", "tcl_wrap_cc")
+
+package(
+    default_visibility = ["//:__subpackages__"],
+    features = ["layering_check"],
+)
+
+filegroup(
+    name = "swig_imports",
+    srcs = glob([
+        "src/swig/tcl/*.i",
+        "src/swig/common/*.i",
+    ]),
+)
+
+cc_library(
+    name = "odb",
+    srcs = glob([
```

Comment:
Longer term, we want to avoid 'glob()`, but for now that things are still a bit coarse-grained it is ok.